addon/components/listbox/-options.hbs

```diff
@@ -0,0 +1,27 @@
+{{#if @isOpen}}
+  <ul
+    id={{@guid}}
+    tabindex="-1"
+    role='listbox'
+    aria-labelledby={{if @hasLabelElement (concat @guid '-label') (concat @guid '-button')}}
+    aria-activedescendant={{@activeOptionGuid}}
+    {{did-insert @registerOptionsElement}}
+    {{will-destroy @unregisterOptionsElement}}
+    {{on 'keypress' @handleKeyPress}}
+    {{on 'keydown' @handleKeyDown}}
+    ...attributes
+    class="pb-1"
```

Comment:
These components shouldn't contain any `class` values; visual styling should be left up to the user!

Reply:
That's alright; I think the test assertion using `.exists()`, as you refactored it to, is fine for our purposes!